.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+{
+  "image": "ghcr.io/swarm-io-internal/docker:latest",
```

Comment:
Umm.. may not appropriate here

Reply:
Gotcha @kapilt! I can remove the `.devcontainer`. I had added it because it makes open source contribution and trying out the package easier and faster.

Reply:
In addition, if you would like me to keep the `devcontainer` but adjust the Docker image, I can create an auto-building Docker image in your org, so you don't have to reference my organization.